test-app/config/ember-try.js

```diff
@@ -10,8 +10,8 @@ module.exports = async function () {
         name: 'minimum-supported',
         npm: {
           devDependencies: {
-            'ember-source': '~3.28.0',
-            'ember-cli': '~3.28.0',
+            'ember-source': '~4.4.0',
```

Comment:
above it says 4.1.0, so it's out of sync

Reply:
tyty